Makefile

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024, The Pallene Developers
+# Pallene Tracer is licensed under the BSD-3-Clause license.
+# Please refer to the LICENSE and AUTHORS files for details
+# SPDX-License-Identifier: BSD-3-Clause 
+
+LUA_INCDIR := /usr/local/include/
```

Comment:
We won't need `INSTALL_DIR` in this case. `INSTALL_INCDIR` would suffice.

Reply:
The idea is that the user is allowed to configure by saying `INSTALL_DIR=/usr/local` instead of being forced to use `INSTALL_DIR=/usr/local/include`. We're used to configuring the base install directory instead of configuring each kind of file separately.

Reply:
I ended up using `INSTALL_DIR` :)